packages/eslint-plugin/tests/rules/prefer-find.test.ts

```diff
@@ -0,0 +1,176 @@
+import { noFormat, RuleTester } from '@typescript-eslint/rule-tester';
+
+import rule from '../../src/rules/prefer-find';
+import { getFixturesRootDir } from '../RuleTester';
+
+const rootDir = getFixturesRootDir();
+
+const ruleTester = new RuleTester({
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    tsconfigRootDir: rootDir,
+    project: './tsconfig.json',
+  },
+});
+
+ruleTester.run('prefer-find', rule, {
+  valid: [
```

Comment:
[Testing] A couple more test cases come to mind:

* Using `[1]` instead of `[0]`
* Using `.at(1)`
* Using `.filter(...)` without a `[0]`
* Putting something between the `.filter(...)` and the `[0]`
* Using another method name, like `.find(...)`

Reply:
added coverage in these areas